deepspeed/pt/deepspeed_zero_optimizer.py

```diff
@@ -81,11 +81,21 @@ def flatten_dense_tensors_aligned(tensor_list, alignment, pg):
     return _flatten_dense_tensors(padded_tensor_list)
 
 
+def get_alignment_padding(tensor_list, alignment):
+    num_elements = sum([tensor.numel() for tensor in tensor_list])
+    remainder = num_elements % alignment
+    return (alignment - remainder) if remainder else remainder
+
+
 def move_to_cpu(tensor_list):
     for tensor in tensor_list:
         tensor.data = tensor.data.cpu()
 
 
+def print_rank_msg(msg):
+    print(f"rank {dist.get_rank()} - {msg}")

```

Comment:
a lot of similar print-related definitions.

Reply:
Will take care of this in a different PR.